scrapers/VRConk.yml

```diff
@@ -0,0 +1,28 @@
+name: "VRConk"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - vrconk.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: //div[@class="box-container"]/div[@class="inner-box-container"]/div[@class="row"]
+    scene:
+      Title: $info//div[contains(@class, 'video-details')]//h1/text()
+      Date:
+        selector: $info//div[@class="stats-container"]/ul/li[3]/span[@class="sub-label"]/text()
```

Comment:
date needs replace and `parsedate` format should be `2006` not `2016`
```yml
        replace:
          - regex: (\d{4}-\d{2}-\d{2})\s.+
            with: $1
        parseDate: 2006-01-02
```

Reply:
Great feedback again.  Will take with me for future scrapers I add.  This is fixed.  Thanks